ansible/roles/shared_dir/tasks/setup_host.yml

```diff
@@ -12,6 +12,20 @@
     - nfs
     - shared_files
 
+- name: Format system data disk
+  sudo: yes
+  filesystem: fstype=ext4 dev={{ datadisk_device }}1
```

Comment:
the first partition on the device is the device name followed by 1. so here the data disk is `/dev/xvde` and its first partition, which is the entire drive in this case is `/dev/xvde1`. the `1` is only added here. when we mount it, its the name without the 1